big_tests/tests/rest_SUITE.erl

```diff
@@ -79,7 +79,7 @@ blank_auth_testcases() ->
 
 test_cases() ->
     [commands_are_listed,
-     non_existent_command_returns404,
```

Comment:
This was not supposed to be deleted, right?

Reply:
its back